qubesappmenus/__init__.py

```diff
@@ -631,8 +643,15 @@ def main(args=None, app=None):
                     parser.error(
                         'this requires --i-understand-format-is-unstable '
                         'and a sacrifice of one cute kitten')
-                sys.stdout.write(''.join('{} - {}\n'.format(*available)
-                    for available in appmenus.get_available(vm)))
+                if not args.fields:
+                    sys.stdout.write(''.join('{} - {}\n'.format(*available)
+                                             for available in
+                                             appmenus.get_available(vm)))
+                else:
+                    for result in appmenus.get_available(
+                            vm, fields=args.fields):
+                        print(' | '.join(result))
```

Comment:
Please make the separate one character. Multi-character field separators may lead to various corner cases (for example how to interpret `a | | b`).

Reply:
pushed a fix